### DISCUSSION TITLE
Long Loading Times and Data Persistence Issues with DB Tables in Budibase

### DISCUSSION THREAD

**Discussion by Abd-Atif** [Budibase/budibase#13092] [Help]
Hello Budibase Community,

I am encountering significant issues with Budibase DB tables, which is currently hosted in Kubernetes pods. The primary problems are as follows:

_Long Loading Times for DB Tables_: On average, it takes more than a minute for any database table to load. This issue persists across multiple attempts and seems to be consistent regardless of the data volume in the tables.

_Data Persistence Issues_: After a table eventually loads and I add some data to it, the data seems to save correctly at first. However, upon refreshing the table, it not only takes a long time to load again but also appears empty, indicating a loss of the newly added data.

To provide a clearer picture of the issue, I've recorded a video demonstrating the problem. 

https://github.com/Budibase/budibase/assets/94058099/723cc64d-5c2c-45a3-99a3-d9f33b076801

Here are some additional details that might help diagnose the issue:

Budibase Version: 2.19.3
Deployment Environment: Kubernetes Pods

I am looking for any advice on how to resolve these issues or if there are known bugs with this version of Budibase that could be causing these problems. Any help or guidance from the community would be greatly appreciated.

Thank you in advance for your assistance.

**Comment by Abd-Atif** [comment]
@jamie-budibase

**Reply by ConorWebb96** [reply]
Hey @Abd-Atif,

Please ensure that sufficient resources are allocated. Additionally, you are a few versions behind; kindly update and retry.

Can you provide information on the resources you have allocated?
